app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+	def new
```

Comment:
Try to be consisten with spaces vs tabs. Configure sublime to use two spaces for indentation.